binrz/man/rizin.1

```diff
@@ -113,6 +113,68 @@ Same as -e bin.usextr=false, do not use extract plugins, useful for dyldcache
 .It Fl z, zz
 do not load strings or load them even in raw
 .El
+.Sh ENVIRONMENT
+.Pp
+_NT_DEBUGGER_EXTENSION_PATH: path(s) to debugger extension DLLs
+.Pp
+ALACRITTY_LOG:               alacritty log file's path
```

Comment:
Remove these:
- `_NT_DEBUGGER_EXTENSION_PATH`
- `ALACRITTY_LOG`
- `COLORTERM`
- `LD_LIBRARY_PATH` and `LIBRARY_PATH`
- `PATH`
- `SHELL`
- `TEMP`
- `TMPDIR`
- `WT_SESSION`
- `TERM`